go.mod

```diff
@@ -7,7 +7,7 @@ toolchain go1.21.1
 require (
 	github.com/docker/buildx v0.13.1 // indirect
 	github.com/docker/cli v26.0.0-rc1+incompatible // indirect
-	github.com/docker/compose/v2 v2.24.7 // indirect
+	github.com/docker/compose/v2 v2.25.0 // indirect
```

Comment:
we have a `make` target now for updating vendoring. It's a long story, but we have replace directives for pinning the version. `make vendor` handles that.

usage:

```
VENDOR_MODULE=github.com/docker/compose/v2@v2.25.0 make vendor
```